rustls/src/crypto/ring/mod.rs

```diff
@@ -14,6 +14,10 @@ use core::fmt;
 
 pub(crate) mod hash;
 pub(crate) mod hmac;
+pub(crate) mod suites;
```

Comment:
In the future, recommend splitting up large commits like these into smaller commits that move one colocated block of code somewhere else.